srcpkgs/xnotify/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'xnotify'
+pkgname=xnotify
+version=0.9.3
+revision=1
+build_style=gnu-makefile
+makedepends="imlib2-devel libXft-devel libX11-devel libXinerama-devel"
+short_desc="Read notifications from stdin and pop them up on the screen"
+maintainer="népéta <nepeta@canaglie.net>"
+license="MIT"
+homepage="https://github.com/phillbush/xnotify"
+distfiles="https://github.com/phillbush/xnotify/archive/refs/tags/v0.9.3.tar.gz"
+checksum=9fa7505ad16c8d5e2305a56d0d88ca3a7e324816b4df7845b920db088482eb24
+
+do_build() {
+	[ -e ${FILESDIR}/config.h ] && cp ${FILESDIR}/config.h config.h
```

Comment:
I don't see anywhere where this is respected by the build process.

Reply:
isn't it? the xnotify makefile brings in `config.h`. (the upstream xnotify no longer has that file, but the latest release still does)

Reply:
That's even more of an argument against adding this capability. Some future release will drop it entirely and we'll be left with vestigial logic that only existed to support custom builds in the first place.

Reply:
> That's even more of an argument against adding this capability. Some future release will drop it entirely and we'll be left with vestigial logic that only existed to support custom builds in the first place.

That's fair, I think. `xnotify` supports Xresources anyway, so I
think it's not as useful as for something like `dmenu`...